Sources/SmithyHTTPAPI/Headers.swift

```diff
@@ -138,15 +138,7 @@ public struct Headers: Sendable {
     }
 
     public func exists(name: String) -> Bool {
-        guard headers.index(of: name) != nil else {
-            return false
-        }
-
-        guard let value = value(for: name) else {
-            return false
-        }
-
-        return !value.isEmpty
+        headers.index(of: name) != nil
```

Comment:
The previous definition of `exists` was wrong because it considered a header with an empty string for its value to "not exist".  Turns out a header with empty string value is valid, so it should definitely "exist".

I suspect that the reason for this logic is that the previous developers changed the definition of "exists" to suit the behavior demanded by empty-header protocol tests, which is wrong and is being fixed by Smithy team.